src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/history.html.twig

```diff
@@ -73,3 +73,5 @@
     {{ form_end(updateOrderStatusForm) }}
```

Comment:
Be careful you changed the permission on this file `100644 → 100755` (same for src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig)
Please reset it to `644` as these files don't need execution permissions